### DISCUSSION TITLE
Automatically squash the mod list based on total height

### DISCUSSION THREAD

**Discussion by CloneWith** [ppy/osu#28994] [Ideas]
When there are few mods in a category, the mod list can be a bit too empty. 

To improve the visual experience, we can set a maximum height for the list column. And the colomn automatically squashes when mods are not enough to fill the maximum height.

Also, it would be better to show the category header at the bottom of a column, in case the list expands (e.g. when we add a new custom mod preset).

![Mods list](https://github.com/user-attachments/assets/c8c13753-2a8c-4184-90d7-104f7095d304)

**Comment by bdach** [comment]
![osu_2024-07-22_10-34-22](https://github.com/user-attachments/assets/1139be3a-1bfe-461a-9f85-53eb38717055)

That's a pass for me. Feels worse imo.

**Reply by CloneWith** [reply]
How about moving the headers down, just next to the bottom bar?

Since we have search & customise bars at the top \(X

**Reply by bdach** [reply]
I'm not even willing to entertain that proposal. It's a completely backwards proposition from the standpoint of UI design. Headers never go on the bottom.
